README.md

```diff
@@ -1,31 +1,93 @@
+`cd script && make migrate-data-small`
 
-A working server application with functional API is required, along with a clear readme.md. Useful and passing tests would be also be viewed favourably
+* This command will add to DB 10 Users and 3 Assets per User for testing reasons
 
-It is appreciated, though not required, if a Dockerfile is included.
+`cd script && make migrate-data-large`
 
-## Submission
+* This command will add to DB 10 Users and 10.000 Assets per User for testing reasons
 
-Just create a fork from the current repo and send it to us!
+Then you can add, edit, remove and get User's favourites like the examples in /examples  
+directory. To generate the needed Bearer token, please call /token endpoint with username  
+& password like in the example
 
-Good luck, potential colleague!
+---
+
+## Makefile Commands
+
+| Command                         | Usage                                                                  |
+|---------------------------------|------------------------------------------------------------------------|
+| start-app                       | `Start app`                                                            |
+| kill-app                        | `Stop app`                                                             |
+| rebuild-app                     | `Rebuild app`                                                          |
+| tests-all                       | `Run both unit and integration tests`                                  |
+| tests-benchmark                 | `Run benchmark tests`                                                  |
+| tests-unit                      | `Run unit tests `                                                      |
+| tests-file FILE={filePath}      | `Run specific file test`                                               |
+| generate-mock FILE={filePath}   | `Generate mock for a specific file`                                    |
+| run-linter                      | `Runs linter`                                                          |
+| generate-swagger-files          | `Generates swagger.json definitions in Docs dir`                       |
+| tests-package PACKAGE={package} | `Run specific package test`                                            |
+| tests-all-with-coverage         | `Run both unit and integration tests via docker with coverage details` |
+| migrate-data-small              | `Run migration of a small dataset Users & Assets in the DB`            |
+| migrate-data-large              | `Run migration of a larger dataset Users & Assets in the DB`           |
+
+* All these are executed through docker containers
+* In order to execute makefile commands type **make** plus a command from the table above
+
+  make {command}
+
+---
+
+## Notes
+
+1. .env is pushed to Git only for the Assessment purpose. Config and .env files should never be tracked.
```

Comment:
Config and .env files usually contain sensitive data that we would not want tracked and seen by everyone in any case. 
Firsty, for our convenience, we could have a dummy file, for example `.env.test`, with dummy values inside, that would be tracked and used for specific reasons (for example integration tests). Or a `.env.local` for the local environment, again with dummy values. 
Anything containing secrets or sensitive data could be in a Password-Secret management tool like Keeper, or Last Pass. Then our collaborators would be given access only to the records that concern them.  
In deployments, the env/config files could be loaded automatically through CI/CD pipelines so that no one has actual access or view to them.